mm2src/mm2_main/src/database.rs

```diff
@@ -78,7 +78,9 @@ async fn migration_2(ctx: &MmArc) -> Vec<(&'static str, Vec<String>)> {
 
 fn migration_3() -> Vec<(&'static str, Vec<String>)> { vec![(stats_swaps::ADD_STARTED_AT_INDEX, vec![])] }
 
-fn migration_4() -> Vec<(&'static str, Vec<String>)> { stats_swaps::add_and_split_tickers() }
+fn migration_4() -> Vec<(&'static str, Vec<String>)> {
+    stats_swaps::do_action_on_stats_db(stats_swaps::ADD_SPLIT_TICKERS)
```

Comment:
This is not clear...

NOTE: `do_action_on_stats_db` is not executing anything, it's only returning a `Vec<(&'static str, Vec<String>)>`

initially, `stats_swaps::add_and_split_tickers` and `stats_swaps::add_coins_prices` in `stats_swap` returns `Vec<(&'static str, Vec<String>)>` so I decided to simplify/refactor them to `stats_swaps::do_action_on_stats_db` (takes an argument (statement to execute))  since both functions literally input/output same result... 

So now instead of calling 
`fn migration_4() -> Vec<(&'static str, Vec<String>)> { stats_swaps::add_and_split_tickers()` 
we can call 
`fn migration_4() -> Vec<(&'static str, Vec<String>)> {
    stats_swaps::do_action_on_stats_db(argument)`

where `argument` can be anything that needs to return `Vec<(&'static str, Vec<String>)>`

Reply:
Oh sure, then it's better to give the `db_common::sqlite::execute_batch` another name like `db_common::sqlite::sql_batch_to_owned` or something like that 
> NOTE: do_action_on_stats_db is not executing anything, it's only returning a Vec<(&'static str, Vec<String>)>

Reply:
It's up to you to do either:
```rust
fn migration_4() -> Vec<(&'static str, Vec<String>)> { stats_swaps::add_and_split_tickers() }

// stats_swaps::add_and_split_tickers()
fn add_and_split_tickers() -> Vec<(&'static str, Vec<String>)> { db_common::sqlite::sql_batch_to_owned(ADD_SPLIT_TICKERS) }
```

or 

```rust
fn migration_4() -> Vec<(&'static str, Vec<String>)> { db_common::sqlite::sql_batch_to_owned(stats_swaps::ADD_SPLIT_TICKERS) }
```